config/locales/en.yml

```diff
@@ -267,7 +272,7 @@ en:
       offender_sar:
         flag_for_disclosure_specialists: "Flag as high profile case?"
         subject_type: "What is their relationship to the MOJ?"
-        third_party: Is this information being requested on someone else's behalf?
+        third_party: Please choose yes or no
```

Comment:
Slightly incongruous change here - is this from Alex?

Reply:
Ah yes so I found a bug in the existing `master` which I completely forgot to mention:

2 keys in the en.yml file were duplicated. This key `third_party` was declared twice but because they were in different parts of the `en.yml` file it was not apparent! I looked through the history and made appropriate amendments.